.config/nextest.toml

```diff
@@ -30,5 +30,11 @@ command = 'cargo run -p crdb-seed --profile test'
 clickhouse-cluster = { max-threads = 1 }
 
 [[profile.default.overrides]]
-filter = 'package(oximeter-db) and test(replicated)'
+filter = 'binary_id(oximeter-db::replicated)'
 test-group = 'clickhouse-cluster'
+
+[[profile.default.overrides]]
+filter = 'binary_id(omicron-nexus::test_all)'
+# As of 2023-01-08, the slowest test in test_all takes 196s on a Ryzen 7950X.
+# 900s is a good upper limit that adds a comfortable buffer.
+slow-timeout = { period = '60s', terminate-after = 15 }
```

Comment:
It's not great, but I've seen tests on my laptop that take up to 1000 seconds -- the saga 'test_action_failure_can_unwind' tests are particularly slow, and definitely need optimizing.

I'm not even sure I'd say this timeout is "wrong" - that test needs fixing - but figured I'd mention it regardless